src/pantry.js

```diff
@@ -1,6 +1,23 @@
 class Pantry {
-  constructor(userIngredients) {
-    this.contents = userIngredients;
+  constructor(userPantry) {
+    this.userIngredients = userPantry
+    // console.log(this.userIngredients);
```

Comment:
This console.log needs to be removed, but glad to see this section is tested and working.